src/components/Hero/index.tsx

```diff
@@ -1,31 +1,56 @@
-import { Box, Grid, Typography } from '@mui/material';
-import * as React from 'react';
-
-import { useAppSelector } from '../../redux/hooks';
+import { Box, Grid, Typography, Button } from "@mui/material";
 
 const Hero = () => {
-  const logoPath = useAppSelector((state) => state.theme.logo);
-
   return (
-    <Box>
+    <Box width="100%" mt={20}>
       <Grid
         container
-        spacing={8}
-        width="100%"
+        spacing={6}
         direction="row"
-        justifyContent="space-between"
         alignItems="center"
-        p={32}
+        justifyContent="space-between"
+        justifyItems="space-between"
       >
-        <Typography variant="h4" component="h4">
-          Lorem ipsum dolor sit amet, <b>consectetur</b>
-          <br />
-          adipiscing elit. Vestibulum feugiat purus <b>urna</b>,
+        <Box
+          width="50%"
+          sx={{
+            display: "flex",
+            justifyContent: "flex-start",
+            justifyItems: "space-between",
+            flexDirection: "column",
+          }}
```

Comment:
I tend to divide styling logic( like sx ={{}} or style ={{}}) from the main business logic(apart from mui codes)
of course it is optional!